ci/apply-Jucer2CMake-reprojucer-to-JUCE-jucers.cmake

```diff
@@ -49,6 +49,8 @@ foreach(jucer_file IN LISTS jucer_files)
     "reprojucer"
     "${JUCE_ROOT}/${jucer_file}"
     "${CMAKE_CURRENT_LIST_DIR}/../cmake/Reprojucer.cmake"
+    "--jucer-version"
+    "${JUCE_VERSION}"
```

Comment:
Note to self: that's worth a copyright year bump

Reply:
Done in https://github.com/McMartin/FRUT/pull/715/commits/6a04e75edcfe687e583281abf15854870d31041c.